Tuist/Dependencies.swift

```diff
@@ -1,24 +1,29 @@
-        .package(url: "https://github.com/tuist/BlueSignals.git", .upToNextMajor(from: "1.0.21")),
-        .package(url: "https://github.com/marmelroy/Zip.git", .upToNextMajor(from: "2.1.1")),
-        .package(url: "https://github.com/rnine/Checksum.git", .upToNextMajor(from: "1.0.2")),
-        .package(url: "https://github.com/stencilproject/Stencil.git", .upToNextMajor(from: "0.14.1")),
-        .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .upToNextMajor(from: "2.8.0")),
-        .package(url: "https://github.com/FabrizioBrancati/Queuer.git", .upToNextMajor(from: "2.1.1")),
-        .package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", .upToNextMajor(from: "1.4.1")),
-        .package(url: "https://github.com/tuist/GraphViz.git", .branch("tuist")),
-        .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMajor(from: "1.0.0")),
-        .package(url: "https://github.com/SwiftGen/SwiftGen", .exact("6.5.0")),
-        .package(url: "https://github.com/kylef/PathKit.git", .upToNextMajor(from: "1.0.0")),
-    ],
+    swiftPackageManager: .init(
+        [
+            .package(url: "https://github.com/tuist/XcodeProj.git", .upToNextMajor(from: "8.5.0")),
+            .package(url: "https://github.com/CombineCommunity/CombineExt.git", .upToNextMajor(from: "1.3.0")),
+            .package(url: "https://github.com/apple/swift-tools-support-core.git", .upToNextMinor(from: "0.2.0")),
+            .package(url: "https://github.com/apple/swift-log.git", .upToNextMajor(from: "1.4.2")),
+            .package(url: "https://github.com/kishikawakatsumi/KeychainAccess.git", .upToNextMajor(from: "4.2.2")),
+            .package(url: "https://github.com/fortmarek/swifter.git", .branch("stable")),
+            .package(url: "https://github.com/tuist/BlueSignals.git", .upToNextMajor(from: "1.0.21")),
+            .package(url: "https://github.com/marmelroy/Zip.git", .upToNextMajor(from: "2.1.1")),
+            .package(url: "https://github.com/rnine/Checksum.git", .upToNextMajor(from: "1.0.2")),
+            .package(url: "https://github.com/stencilproject/Stencil.git", .upToNextMajor(from: "0.14.1")),
+            .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .upToNextMajor(from: "2.8.0")),
+            .package(url: "https://github.com/FabrizioBrancati/Queuer.git", .upToNextMajor(from: "2.1.1")),
+            .package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", .upToNextMajor(from: "1.4.1")),
+            .package(url: "https://github.com/tuist/GraphViz.git", .branch("tuist")),
+            .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMajor(from: "1.0.0")),
+            .package(url: "https://github.com/SwiftGen/SwiftGen", .exact("6.5.0")),
+            .package(url: "https://github.com/kylef/PathKit.git", .upToNextMajor(from: "1.0.0")),
+        ],
+        productTypes: [
+            "PathKit": .framework,
+        ]
```

Comment:
![image](https://user-images.githubusercontent.com/13840545/150909895-8556f6a3-fb38-4e6c-882f-f6859b9c58af.png)

Reply:
That's the only warning we get regarding this issue. TuistGenerator does import PathKit which makes sense given the warning ([see here](https://github.com/tuist/tuist/blob/main/Sources/TuistGenerator/Extensions/AbsolutePath+Extras.swift)).

However, this seems like a duplicate since that same code can exist [here](https://github.com/tuist/tuist/blob/main/Sources/TuistSupport/Extensions/AbsolutePath+Extras.swift)  unless it can't for some reason in which case we need tom make `PathKit` dynamic